token/src/instruction.rs

```diff
@@ -68,6 +85,7 @@ pub enum TokenInstruction {
     ///   0. `[signer]` Current owner of the token or account.
     ///   1. `[writable]` token or account to change the owner of.
     ///   2. `[]` New owner
```

Comment:
Document that the new owner can be a `Multisig` account?

Reply:
Yeah, I debated that, the `InitializeMultisig` instruction documentation mentions that the multisig can take the place of any owner, I was avoiding adding a bunch of duplicate verbiage.  I'm not opposed to adding it if you think it's worth it.

Reply:
Ah, I guess we just need one location that says, "owner could be either a multisig address, in which case M of the N signers must be present, or any other address where the matching signer must be present"